src/components/TileComponents.js

```diff
@@ -0,0 +1,29 @@
+import { render } from '../shared/dom';
+
+export const TileComponent = ({ title, image }) => {
+    return `
+    <div class="movie-tile">
```

Comment:
Also I remember that we were talking about this "hover" effect on Tile, when we hover over and we see some details about the certain asset, this should be implemented in scope of this task too

Reply:
hover effect has been added, now after hover on movie image we can see movie details. I need to know whether I should style it now or later?

Reply:
hover effect has been added, now after hover on movie image we can see movie details. I need to know whether I should style it now or later.

Reply:
Hover is working